### DISCUSSION TITLE
Getting Started W Artisan and Kaleido

### DISCUSSION THREAD

**Discussion by asarule** [artisan-roaster-scope/artisan#1467] [Ideas]
Is there a reference source for Artisan roast profile/bean/machine combinations? A searchable online database? The Aillio Bullet community has a database of user supplied roast/bean combinations for the their single machine platform in https://roast.world/roasts.   It is a feature that I believe greatly enhances the utility of the Aillio Bullet ecosystem. I appreciate that the Artisan community has roaster platform diversity, and therefore another layer of complexity above the single roaster platform world of the Aillio Bullet world, never-the-less, as a new to Artisan roaster struggling with finding profile/bean/machine combinations that produce satisfying results, such a resource would be greatly appreciated and help reduce some painful and expensive wasted failed roast tuition costs.

**Comment by CarefreeBuzzBuzz** [comment]
It has been tried in the past, and maybe others can fill you in on that. The Artisan team has no resources to do such a thing currently___________________________Michael Herbert, CarefreeBuzzBuzzOn Feb 26, 2024, at 8:12 AM, asarule ***@***.***> wrote:﻿
Is there a reference source for Artisan roast profile/bean/machine combinations? A searchable online database? The Aillio Bullet community has a database of user supplied roast/bean combinations for the their single machine platform in https://roast.world/roasts.   It is a feature that I believe greatly enhances the utility of the Aillio Bullet ecosystem. I appreciate that the Artisan community has roaster platform diversity, and therefore another layer of complexity above the single roaster platform world of the Aillio Bullet world, never-the-less, as a new to Artisan roaster struggling with finding profile/bean/machine combinations that produce satisfying results, such a resource would be greatly appreciated and help reduce some painful and expensive wasted failed roast tuition costs.

**Comment by prmango43** [comment]
I don't see a follow-up, but just in case there is a roastetta.com for Kaleido/others.
